docs/docs/writing-stories.mdx

```diff
@@ -143,29 +144,14 @@ feedback.
 Make sure to combine the dumped stories and NLU examples with your original
 training data for the next training.
 
-## Visualization of conversations
-
-During the interactive learning, Rasa will plot the current conversation
-and a few similar conversations from the training data to help you
-keep track of where you are.
-
-You can view the visualization at [http://localhost:5005/visualization.html](http://localhost:5005/visualization.html)
-as soon as you’ve started interactive learning.
-
-To skip the visualization, run `rasa interactive --skip-visualization`.
-
-
-
-<img alt="image" src={useBaseUrl("/img/interactive_learning_graph.gif")} />
```

Comment:
it looks like we're not using the file anymore, so we can remove it from the `static/` folder?